CHANGELOG.md

```diff
@@ -3,8 +3,6 @@ All notable changes to this project will be documented in this file - [read more
 
 ## [UNRELEASED]
 
-## [0.11.0]
```

Comment:
Should this be here ?

Reply:
Let me add back, merging error probably :(